docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md

```diff
@@ -0,0 +1,32 @@
+# Forcible terminate instances if the job is stuck in umount
+
+Test verifies that if a run is not stopped after a certain period of time defined for run by CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN parameter the underlying node is terminated
+
+**Prerequisites**:
+- Non-admin user
+
+| Steps | Actions | Expected results |
+|:-----------------------------------------------------------------------------------------:|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | Open the **Tools** page | |
+| 3 | Select test tool | |
+| 4 | At the tool page, hover over the "**Run v**" button | | 
+| 5 | Click the "**Custom settings**" button in the list | |
+| 6 | Expand the **ADVANCED** section | |
+| 7 | Input `echo "#!/usr/bin/env bash" > /usr/local/sbin/umount && echo "sleep infinity" >> /usr/local/sbin/umount && chmod +x /usr/local/sbin/umount` command into the ***Cmd template*** field | |
+| 8 | Launch tool | |
+| 9 | At the **Runs** page, click the just-launched run | | 
+| 10 | Open run's *Console* | |
+| 11 | Wait until `Unmounting all storage mounts` appears in the log | |
+| 12 | Wait `CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN` default time (1min) | Run isn't stopped automatically |
+| 13 | Repeat steps 3-7 | |
+| 14 | Hover over **v** button next to "Add parameter" | |
+| 15 | Click on ***String parameter*** item | |
+| 16 | Input `CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN` parameter Name and `2` as parameter Value | |
+| 17 | Repeat steps 8-11 | |
+| 18 | Wait until ***CleanupEnvironment*** task appears | |
+| 19 | Click ***CleanupEnvironment*** task | `Will wait for 2min to let the run stop normally. Otherwise it will be terminated.` is shown in the log |
+| 20 | Wait `CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN` time specified at the step 16 | <li> Run is stopped <li> `Run <runID> is still running after 2min. Terminating a node.` is shown in the log, where `<runID>` is ID run launched at step 17 |
+
+**After:**
+- Stop the runs launched at steps 9 and 17
```

Comment:
Fix to `Stop the run launched at step 8`.
As for the run launched at step 17 - it was stopped at step 20, isn't it?